source/standalone/workflows/rl_games/train.py

```diff
@@ -130,6 +135,19 @@ def main():
     )
     env_configurations.register("rlgpu", {"vecenv_type": "IsaacRlgWrapper", "env_creator": lambda **kwargs: env})
 
+    if args_cli.checkpoint is not None:
+        resume_path = retrieve_file_path(args_cli.checkpoint)
+        agent_cfg["params"]["load_checkpoint"] = True
+        agent_cfg["params"]["load_path"] = resume_path
+        print(f"[INFO]: Loading model checkpoint from: {agent_cfg['params']['load_path']}")
+
+    if args_cli.sigma is not None:
+        train_sigma= float(args_cli.sigma)
+    else:
+        train_sigma = None
+    
+
+
```

Comment:
Please remove unnecessary white-space

Reply:
fixed it now :-)
https://github.com/sizsJEon/IsaacLab/blob/0843442802ed47b30bcd4bf6978af611abd33411/source/standalone/workflows/rl_games/train.py#L146-L148

sorry for delay, there was a issue about my ide, so I had to use terminals.